static/javascripts/components/profile.jsx

```diff
@@ -66,25 +60,21 @@ getInitialState: function(){
 },
 
     render: function(){
-    console.log(user_data)
-    return(
-        <div className="mdl-card mdl-shadow--2dp">
-            <div className="mdl-card__title mdl-color--primary">
-                <h2 className="mdl-card__title-text">User Info</h2>
-            </div>
-            <div className="mdl-card__supporting-text">
-            Username: {user_data[0].username} <br/>
-            User Type: {user_data[0].primary_affiliation} <br/>
-            Email: {user_data[0].email} <br/>
-            Birth Date: {user_data[0].dob} <br/>
-            Gender: {user_data[0].gender} <br/>
-            Ethnicity: {user_data[0].ethnicity} <br/>
-            Native Language: {user_data[0].native_language} <br/>
-            Status: {this.state.status} <br/>
-            Courses Enrolled: {this.state.courses} <br/>
-            Departments: {this.state.departments} <br/>
-            </div>
-        </div>
+    var style = {
+        listStyleType: "none",
+        fontSize: "20px"
+    };
+
+        return(
+        <ul style={style}>
+            <li>Username: {user_data[0].username}</li>
+            <li>Email: {user_data[0].email}</li>
+            <li>Birth Date: {user_data[0].dob}</li>
+            <li>Gender: {user_data[0].gender}</li>
+            <li>Ethnicity: {user_data[0].ethnicity}</li>
+            <li>Native Language: {user_data[0].native_language}</li>
+            <li>Year: {this.state.status}</li>
+        </ul>
     );
     }
-});
\ No newline at end of file
+});
```

Comment:
So I'm currently working on this file. I will directly make your changes into my branch. I just don't want merge conflicts on my end again, so could you not edit this file?